keyboards/ergodone/config.h

```diff
@@ -22,7 +22,7 @@
 #define MOUSEKEY_MAX_SPEED      7
 #define MOUSEKEY_WHEEL_DELAY 0
 
-#define TAPPING_TOGGLE  1
+#define TAPPING_TOGGLE  2
```

Comment:
We shouldnt really be modifying another keyboard as part of this PR
```suggestion
#define TAPPING_TOGGLE  1
```

Reply:
Oops my bad, now you can tell which other keyboard I have 😁